docs/root/configuration/listeners/network_filters/zookeeper_proxy_filter.rst

```diff
@@ -31,8 +31,9 @@ in the configuration snippet below:
 Statistics
 ----------
 
-Every configured ZooKeeper proxy filter has statistics rooted at *<stat_prefix>.zookeeper.*. The
-following counters are available:
+Every configured ZooKeeper proxy filter has statistics rooted at *<stat_prefix>.zookeeper.*.
+*_resp_fast* and *_resp_slow* metrics will only be emitted when enabling the latency threshold metrics.
```

Comment:
slight re-wording suggestion: 

will only be emitted when ``enable_latency_threshold_metrics`` is set to `true`.